frontend/src/components/Notification/index.js

```diff
@@ -1,24 +1,55 @@
 import '@/components/Notification/index.scss';
 
 import user from '@/assets/user.svg';
+import ago from '@/commons/utils/ago';
+import { ACTION_MAPPER } from '@/constants/mapper';
 import Component from '@/libs/Component';
 
 class Notification extends Component {
   constructor($container, initialState) {
     super($container, initialState);
-
     this.render();
   }
 
+  setAgo(createdAt) {
+    return ago(createdAt);
+  }
+
+  setText(todoLogData) {
+    const obj = {};
+    const { action } = todoLogData;
+    Object.entries(todoLogData).forEach((kv) => Object.assign(obj, { [kv[0]]: `<b>${kv[1]}</b>` }));
+    const { prevTitle, nextTitle, prevType, nextType } = obj;
+
+    switch (action) {
+      case 'add':
+        return `${nextType}에 ${nextTitle}을/를 ${ACTION_MAPPER[action]}하였습니다.`;
+      case 'delete':
+        return `${nextTitle}을/를 ${ACTION_MAPPER[action]}하였습니다.`;
+      case 'update':
+        return `${prevTitle}을/를 ${nextTitle}(으)로 ${ACTION_MAPPER[action]}하였습니다.`;
+      case 'move':
+        return `${nextTitle}을/를 ${prevType}에서 ${nextType}으로 ${ACTION_MAPPER[action]}하였습니다.`;
+      default:
+        `알 수 없는 오류`;
+        break;
```

Comment:
```suggestion
        return;
```

얘도 return; 문으로 통일하는 것도 좋아보임니다!